test/support/jobs.rb

```diff
@@ -15,11 +15,12 @@ def each_iteration(omg)
   end
 end
 
-class TimeCursorJob < ActiveJob::Base
+class UnserializableCursorJob < ActiveJob::Base
   include JobIteration::Iteration
+  UnserializableCursor = Class.new
```

Comment:
Since we're getting all of `ActiveJob`'s serialization now, we have to explicitly create an object that cannot be serialized. `BasicObject` doesn't work though, because that straight up blows up `ActiveJob` because it doesn't respond to `respond_to?` 😅